deployment/k8s-deploy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Not sure where you were planning to apply the parameters you require for the env_file.
Perhaps here it can be a good location too?
(In other words, I mean the env_file should now contain the sensitive info in the git but should be populated during deployment.)

Reply:
The docker container executes an `envsubst` on container startup.  Only blank values should appear in the relevant files on github.